webapp/channels/src/components/pdf_preview.tsx

```diff
@@ -215,7 +239,7 @@ export default class PDFPreview extends React.PureComponent {
         if (!this.state.success) {
             return (
                 <FileInfoPreview
-                    fileInfo={this.props.fileInfo}
+                    fileInfo={this.props.fileInfo as FileInfo}
```

Comment:
This should already be a FileInfo. Why is the casting needed?

Reply:
fixed